homeassistant/components/litejet/scene.py

```diff
@@ -59,7 +63,17 @@ def _on_connected_changed(self, connected: bool, reason: str) -> None:
         self.async_write_ha_state()
 
     @property
-    def extra_state_attributes(self):
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, f"{self._entry_id}_mcp")},
+            name="LiteJet",
+            manufacturer="Centralite",
+            model="CL24",
```

Comment:
This conflicts with what's set in in `light.py` and `switch.py`.
Or does a centralite device which generates scene entities not have any light or switch entities?